sequencing-server/package-lock.json

```diff
@@ -1642,6 +1642,17 @@
         "npm": "1.2.8000 || >= 1.4.16"
       }
     },
+    "node_modules/body-parser/node_modules/qs": {
```

Comment:
Do we need this lock file change?

Reply:
Bummer that body-parser still uses qs despite URLSearchParams for a long while now.

https://dev.to/nerdyman/replacing-query-string-with-native-urlsearchparams-4kdg